### PR TITLE
[BACKLOG-12634] Mavenize pdi-palo-core

### PR DESCRIPTION
**Pull request by Kyrrui**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<parent>
+  <groupId>org.pentaho</groupId>
+  <artifactId>pentaho-ce-jar-parent-pom</artifactId>
+  <version>2.0.4</version>
```
Comment (tmcsantos) [inline pom.xml]: this version is too old, should be 8.0-SNAPSHOT

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <packaging>jar</packaging>
+  <scm>
+    <developerConnection>scm:git:git@github.com:pentaho/pdi-palo-core.git</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+  <dependencies>  
+  <dependency>
+        <groupId>pentaho-kettle</groupId>
+        <artifactId>kettle-core</artifactId>
+        <version>8.0-SNAPSHOT</version>
```
Comment (tmcsantos) [inline pom.xml]: dependencies versions should be defined as properties, in this special case define a `kettle.version` property

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        <scope>provided</scope>
+ </dependency>  
+  <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.16</version>
```
Comment (tmcsantos) [inline pom.xml]: add `log4j.version`property

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency> 
+  <dependency>
+      <groupId>palo</groupId>
+      <artifactId>palojlib</artifactId>
+      <version>1.0.73</version>
```
Comment (tmcsantos) [inline pom.xml]: change to property

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency> 
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
```
Comment (tmcsantos) [inline pom.xml]: change to property

**Review by tmcsantos — CHANGES_REQUESTED** [review]

**Review by smaring — COMMENTED** [review]
just another tip:  by convention, we usually work on feature branches named after the ticket we are using, in this case BACKLOG-12634, not master.  This makes it easier for others to add your remote, check out your branch, and not have to worry about confusing it with the upstream pentaho master

**Comment by smaring** [comment]
Also, you need to delete build-res, libext, .classpath, build.properties, build.xml, ivy.xml, and ivysettings.xml.  Leave no trace of the prior build setup.

**Comment by smaring** [comment]
other than that, and what Tiago commented, it looks like the resultant jar is in good shape by comparison https://image.prntscr.com/image/6Ffu7g1uRC2ciwwFRM2a7g.png

**Review by tmcsantos — APPROVED** [review]

**Pull request merged**